src/database.ts

```diff
@@ -1476,6 +1491,54 @@ class Database extends common.GrpcServiceObject {
     return metadata.state || undefined;
   }
 
+  /**
+   * Get this database's dialect to retrieves the dialect of the database
+   *
+   * @see {@link #getMetadata}
+   *
+   * @method Database#getDatabaseDialect
+   *
+   * @param {object} [gaxOptions] Request configuration options,
+   *     See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions}
+   *     for more details.
+   * @param {GetDatabaseDialectCallback} [callback] Callback function.
+   * @returns {Promise<EnumKey<typeof, databaseAdmin.spanner.admin.database.v1.DatabaseDialect> | undefined>}
+   *     When resolved, contains the database dialect of the database if the dialect
+   *     is defined.
+   */
+
```

Comment:
Add an example in the documentation like we have in get state method



   ```
   @example
   * const {Spanner} = require('@google-cloud/spanner');
   * const spanner = new Spanner();
   * const instance = spanner.instance('my-instance');
   * const database = instance.database('my-database');
   * const dialect = await database.getDialect();
   * const isGoogleSQL = (dialect === 'GOOGLE_STANDARD_SQL');
   ```

Reply:
Done